### ISSUE TITLE
Facebook Page Proof

### ISSUE THREAD

**Issue by mauvehed** [keybase/client#8543]
How does one prove themselves on a Facebook Page? Not your personal profile, but a Page you're an admin of? If I use the CLI to generate the URL I'm supposed to click, it directs to me to my personal profile and not my Page. The payload is also not something I can copy/paste into my Page so I'm not sure what the contents should look like. Is there a way to have the CLI output just the payload and not generate a URL?

**Comment by necrose99** [comment]
yup facebook is seeing it as spam and not to your wall by page id.... by the look of it....

SPAM-API problly killzing it.

**Comment by maxtaco** [comment]
cc: @oconnor663

**Comment by oconnor663** [comment]
Adding a flag on the command line to just dump the proof into the terminal is a good idea. Doing that much shouldn't be hard.

The tricky part will be to make sure that the client can _verify_ these proofs once they land in the page. I think this has worked in the past, but we haven't tested it in a while, and in general the verification code is super duper sensitive to the details of page layout.
